critiquebrainz/frontend/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -3,28 +3,32 @@
 # This file is distributed under the same license as the PROJECT project.
 # 
 # Translators:
+# jesus2099, 2020
+# PATATE12, 2020
```

Comment:
I don't know if you would like cleaning this up but you can remove PATATE12, which is same as jesus2099. :)

Reply:
This text is automatically generated by transifex when we download the translations, so I'd prefer to not manually make changes. It's possible we can update it in transfex to make it look better, otherwise it'll reappear the next time we update this file